### PR TITLE
Feat react components

### PR DESCRIPTION
**Pull request by lee-philip-n**
Trello implementation: https://trello.com/c/qUfTS3Zu

### CODE REVIEW

File: client/data/dummyData.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  { name: 'Namson Leung', img: '../dist/pictures/Namson Leung.jpg' },
+  { name: 'Leung Namson', img: '../dist/pictures/Leung Namson.jpg' },
+  { name: 'Nicholas Peyrichou', img: '../dist/pictures/Nicholas Peyrichou.jpg' },
+  { name: 'Peyrichou Nicholas', img: '../dist/pictures/Peyrichou Nicholas.jpg' },
+  { name: 'Sara Brandes', img: '../dist/pictures/Sara Brandes.jpg' },
+  { name: 'Brandes Sara', img: '../dist/pictures/Brandes Sara.jpg' },
+  { name: 'Shaital Desai', img: '../dist/pictures/Shaital Desai.jpg' },
+  { name: 'Desai Shaital', img: '../dist/pictures/Desai Shaital.jpg' },
+  { name: 'Wayman Ng', img: '../dist/pictures/Wayman Ng.jpg' },
+  { name: 'Ng Wayman', img: '../dist/pictures/Ng Wayman.jpg' },
```
Comment (josh-alamillo7) [inline client/data/dummyData.js]: I'd rename this file to "actorData.js" for clarity

File: tests/componentsUnitTests.test.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import App from '../client/src/components/App';
+import CrewsCastsBox from '../client/src/components/CrewsCastsBox';
+import CrewsCastsImgEntry from '../client/src/components/CrewsCastsImgEntry';
+import CrewsCastsNameEntry from '../client/src/components/CrewsCastsNameEntry';
+import dummyData from '../client/data/dummyData';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe('App components', () => {
+  it('Component CrewsCastsBox should exists', () => {
```
Comment (josh-alamillo7) [inline tests/componentsUnitTests.test.jsx]: maybe rephrase this to "should contain a component named CrewsCastsBox"

**Comment by lee-philip-n** [comment]
fixed the issues josh mentioned and ready for review again!

**Comment by Angelhendrix** [comment]
Looks good Philip
